### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: examples/trivial/index.js [hunk 1]
```diff
@@ -24,7 +24,52 @@ var React = require("react");
 // maps cannot pass through two stages).
 var MatrixReactSdk = require("../../src/index");
 
-React.render(
-    <MatrixReactSdk.MatrixChat />,
+// Here, we do some crude URL analysis to allow
+// deep-linking. We only support registration
+// deep-links in this example.
+function routeUrl(location) {
+    if (location.hash.indexOf('#/register') == 0) {
```
Comment (ara4n) [inline examples/trivial/index.js]: What's the rationale for #/foo URLs here (other than copying angular)?

File: src/controllers/molecules/ServerConfig.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ module.exports = {
         return {
             onHsUrlChanged: function() {},
             onIsUrlChanged: function() {},
-            default_hs_url: 'https://matrix.org/',
+            default_hs_url: 'http://localhost:8008',
```
Comment (ara4n) [inline src/controllers/molecules/ServerConfig.js]: mis-commit i guess?

**Comment by ara4n** [comment]
lgtm on cursory overview other than comments above, assuming it's tested and works :)

File: examples/trivial/index.js (hunk 1 above)
Comment (dbkr) [inline examples/trivial/index.js]: It's the example, so just did the simplest thing possible.

**Pull request merged**